packages/itwin/tree-widget/src/components/trees/category-tree/Categories.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "../../../../node_modules/@itwin/presentation-common/Ruleset.schema.json",
+  "id": "ui-framework/CategoriesTree",
```

Comment:
should this id now be updated to reflect where this tree is coming from? tree-widget/CategoriesTree?

Reply:
Yes it makes sense to update id to match package. 

I also was thinking to update css classes to use `tree-widget` prefix instead of `ui-fw` but I am worried that it might be bigger breaking change than moving these trees from one package to another (apps that override styles of those tree will need to update css instead of just updating import). What do you think?

Reply:
yea i was going to comment about the use of ui-fw, but i assumed this was mostly a direct port of the code from the core repo. i know the appui team is planning to kill off all usage of uifw and buic scss vars/mixins for appui 4.0 wasnt sure if we should tackled the ui thing then. thx for bringing this up

Reply:
Technically the components in this repo are different components than the ones currently in core. That, and the fact that people overriding these styles is unlikely, IMO, suggests we should go ahead and rename the classes to match the new package.

Reply:
Renamed classes that contained `ui-fw` prefix in their name